### PR TITLE
New Releases Tx v3.1.3, Block v3.2.1, VM v5.3.1

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

**Comment by holgerd77** [comment]
I would want to release this latest tomorrow morning, so optimally directly push on review if you have got change requests.

**Comment by holgerd77** [comment]
Client CI failure is unrelated and can be ignored, in doubt I will merge by admin-merge if necessary

**Review by ryanio — APPROVED** [review]
lgtm!

File: packages/client/karma.conf.js [hunk 1]
```diff
@@ -14,7 +14,17 @@ module.exports = function (config) {
...
+            // you may ask why on earth do we need this...,
+            // so this is to make sure `cjs` extensions are treated as actual scripts and not text files
+            // https://github.com/monounity/karma-typescript/blob/master/packages/karma-typescript/src/bundler/bundle-item.ts#L18 does not have cjs extension listed, so our file is not treated as script, and eventually require-ing it leads to a typeerror, since we get a string instead
+            // luckily it's an OR with rhs being `this.transformedScript` expression, so all we need to do is to set it to true (which we do below)
+            if (context.module.includes('web-encoding')) {
+              // needed to set a flag transformedScript on BundledItem described above, https://github.com/monounity/karma-typescript/blob/master/packages/karma-typescript/src/bundler/transformer.ts#L94
+              return callback(0, { dirty: true, transformedScript: true });
+            }
+            return callback(0, false);
+          },
```
Comment (holgerd77) [inline packages/client/karma.conf.js:27]: Tool magician! 😋  🥳

**Comment by holgerd77** [comment]
Just published the following releases on npm:

- `@ethereumjs/tx` [v3.1.3](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftx%403.1.3)
- `@ethereumjs/block` [v3.2.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fblock%403.2.1)
- `@ethereumjs/vm` [v5.3.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fvm%405.3.1)

The following releases have been deprecated on npm:

- `@ethereumjs/tx` [v3.1.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftx%403.1.1)
- `@ethereumjs/tx` [v3.1.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftx%403.1.2)

**Pull request merged**